### PR TITLE
There is typos error "reproducable" instead of that it should be "rep…

### PR DESCRIPTION
**Pull request by niranjansosvns**
…roducible" I guess

Please verify the above changes

### CODE REVIEW

**Comment by KenanY** [comment]
Previously: https://github.com/npm/docs/pull/717.

File: content/getting-started/using-a-package.json.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ A `package.json` file affords you a lot of great things:
 1. It serves as documentation for what packages your project depends on.
 2. It allows you to specify the versions of a package that your project
 can use using [semantic versioning rules][1].
-3. Makes your build reproducable which means that its *way* easier
+3. Makes your build  reproducible which means that its *way* easier
```
Comment (ashleygwilliams) [inline content/getting-started/using-a-package.json.md]: you have an extra space in here! if you remove it i'll be happy to merge

**Comment by ashleygwilliams** [comment]
looks like the author of #717 closed their PR, @KenanY,  so it's yours for the taking @niranjansosvns - just need that extra space removed :)

**Comment by ashleygwilliams** [comment]
closed because of lack of response.

**Pull request closed without merging**